pkg/cluster/cluster.go

```diff
@@ -36,8 +36,8 @@ type Peer struct {
 }
 
 const (
-	defaultPushPullInterval = 5 * time.Second
-	defaultGossipInterval   = 5 * time.Second
+	DefaultPushPullInterval = "5s"
+	DefaultGossipInterval   = "5s"
```

Comment:
String constants here just because the `main` consumer cannot understand proper typed ones seems hacky.

Reply:
`time.Duration` has a `String()` method even.